CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### 0.11.2 (Next)
 
+* [#105](https://github.com/slack-ruby/slack-ruby-bot-server/pull/105): Update README with OAuth code grant detail - [@jcraigk](https://github.com/jcraigk).
```

Comment:
No CHANGELOG update for docs. Btw, if you want a project we want danger-changelog to ignore markdown files, see https://github.com/dblock/danger-changelog/issues/14.